src/adagrams.js

```diff
@@ -1,15 +1,145 @@
+const HAND_SIZE = 10;
+
+const LETTER_BANK = {
+  A : 9,
+  N : 6,
+  B : 2,
+  O : 8,
+  C : 2,
+  P : 2,
+  D : 4,
+  Q : 1,
+  E : 12,
+  R : 6,
+  F : 2,
+  S : 4,
+  G : 3,
+  T : 6,
+  H : 2,
+  U : 4,
+  I : 9,
+  V : 2,
+  J : 1,
+  W : 2,
+  K : 1,
+  X : 1,
+  L : 4,
+  Y : 2,
+  M : 2,
+  Z : 1
+};
+
+const SCORES_CHART = {
+  'A': 1, 'E': 1, 'I': 1, 'O': 1, 'U': 1, 'L': 1, 'N': 1, 'R': 1, 'S': 1, 'T': 1,
+  'D': 2, 'G': 2,
+  'B': 3, 'C': 3, 'M': 3, 'P': 3,
+  'F': 4, 'H': 4, 'V': 4, 'W': 4, 'Y': 4,
+  'K': 5,
+  'J': 8, 'X': 8,
+  'Q': 10, 'Z': 10
+};
+
+const BONUS_MIN_LENGTH = 7;
+const BONUS_POINTS = 8;
+
+const createLetterPool = () => {
+  let letterPool = [];
```

Comment:
Since we never reassign letterPool, we just modify it, we should use `const`. Whenever possible, we should prefer `const` over `let`

```suggestion
  const letterPool = [];
```